src/particles/feedback_CIC_gpu.cu

```diff
@@ -36,7 +36,7 @@ int snr_n;
 }  // namespace supernova
 
   #ifndef O_HIP
-__device__ double atomicMax(double* address, double val)
+__device__ double Atomic_Max(double* address, double val)
```

Comment:
HIP has `atomicMax`, so if compiling with O_HIP, we use it.  Changing this will break on AMD if you also update client code to call the new name.

Reply:
Resolved in [1465895](https://github.com/helenarichie/cholla/commit/146589509a5250e3280c3212889c52a31ed513c1)